embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/IntegrationTestRule.kt

```diff
@@ -133,6 +133,16 @@ internal class IntegrationTestRule(
         Embrace.getImpl().stop()
     }
 
+    fun runTest(
+        setupAction: Harness.() -> Unit,
+        testCaseAction: Harness.() -> Unit,
+        assertAction: Harness.() -> Unit,
```

Comment:
Right now these all return `Harness`, but I think in future it would be better for them to return more specific types. Setup action should act on a type that alters configuration, the test case action should act on Embrace/a shim of that interface, and assert action should act on a type from the delivery layer